weave/tests/test_weave_client.py

```diff
@@ -305,8 +305,9 @@ def test_calls_query(client):
                 "os_version": platform.version(),
                 "os_release": platform.release(),
                 "sys_version": sys.version,
-            },
+            }
         },
+        summary={},
```

Comment:
seems odd to me that we now have an empty dict where we didn't before - is this intentional?